ListSum/ListSum.py

```diff
@@ -0,0 +1,54 @@
+def get_list_one_by_one(lst):
+    """ get list of numbers, strings are ignored. return: list of numbers """
+    print('Type a number, type "stop" to end the list')
+    member = input()
+    if(member == "stop"):
+        return lst
+    else:
+        if(member.isdigit()):
+            lst.append(int(member))
+        else:
+            print("{} is ignored, not a number".format(member))
+        return get_list_one_by_one(lst)
```

Comment:
no need for recursion here, this is much less readable than just using a `while` loop. I understand that you are just trying to show off, but really, don't use recursion unless it is absouloutly necessary.